lib/MusicBrainz/Server/Form/Filter/ReleaseGroup.pm

```diff
@@ -46,18 +47,18 @@ sub options_artist_credit_id {
 
 sub options_secondary_type_id {
     my ($self, $field) = @_;
-    return [
-        map +{ value => $_->id, label => $_->l_name },
-        @{ $self->secondary_types }
-    ];
+    my @types = ({ value => 999, label => lp('[none]', 'release group type') });
+    push (@types, map +{ value => $_->id, label => $_->l_name },
+                  @{ $self->secondary_types });
+    return \@types;
```

Comment:
I think it'd be more self-documenting to use `[none]` for the value (doesn't have to be a number).

Also, you can just put the new element at the beginning of the existing list:

```Perl
    return [
        { value => '[none]', label => lp('[none]', 'release group type') },
        map +{ value => $_->id, label => $_->l_name },
        @{ $self->secondary_types }
    ];
```

this is easier to read and makes the diff much smaller